src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card_imac">
+        <img
+          class="card_imac_img"
+          src="../src/images/imac.jpeg"
+          alt="imac_photo"
+        />
+      </div>
+      <div class="card_info">
+        <h1 class="card_text_name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+        <h2 class="card_text_code">Product code: 195434</h2>
+        <div class="card_stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <h2 class="stars_number">Reviews: 5</h2>
+        </div>
+        <div class="card_price">
+          <h2 class="card_price_name">Price:</h2>
```

Comment:
The class name 'card_price_name' does not follow BEM methodology. Consider using 'card__price-name' for an element within 'card'.